samcli/commands/_utils/options.py

```diff
@@ -571,6 +587,21 @@ def image_repositories_option(f):
     return image_repositories_click_option()(f)
 
 
+def remote_invoke_parameter_click_option():
+    return click.option(
+        "--parameter",
+        multiple=True,
+        type=RemoteInvokeBotoAPIParameterType(),
+        callback=remote_invoke_boto_parameter_callback,
+        required=False,
+        help="Additional parameters for the boto API call",
```

Comment:
Should we add name of the APIs so that customers can find other keys of the specific API?